.github/CODEOWNERS

```diff
@@ -9,5 +9,5 @@
 #  More info: https://help.github.com/articles/about-codeowners/
 #
 
-* @YuliaTsareva @MarsiBarsi @waterplea @Tom910 @SuperOleg39 @dekhanov @splincode @IKatsuba
+* @YuliaTsareva @MarsiBarsi @waterplea @SuperOleg39 @dekhanov @splincode @IKatsuba
```

Comment:
@IKatsuba please review ))

Reply:
@alexkvak 
<img width="452" alt="image" src="https://user-images.githubusercontent.com/12021443/230871559-9e4c3413-b350-4ad1-a8f9-9fbe988cdf5c.png">

Reply:
https://github.com/dekhanov

/shrug

Reply:
why is he not added to the organization then?